plone/app/upgrade/v43/configure.zcml

```diff
@@ -241,6 +241,21 @@
             description=""
             handler="..utils.null_upgrade_step"
             />
+
+    </genericsetup:upgradeSteps>
+
+    <genericsetup:upgradeSteps
+        source="4313"
+        destination="4314"
+        profile="Products.CMFPlone:plone">
+
+        <genericsetup:upgradeDepends
+            title="Add default search options"
+            description="Add sort_on field to search controlpanel."
+            import_steps="propertiestool"
```

Comment:
This applies the entire `propertiestool.xml` file from CMFPlone! That is far too much, overriding changes made by managers. The right thing would be to do this with a few lines of python that only add this field if it is not there yet.